packages/cashier/src/components/resend-email-button-wrapper/__tests__/resend-email-button-wrapper.spec.tsx

```diff
@@ -1,40 +1,59 @@
 import React from 'react';
 import { fireEvent, render, screen } from '@testing-library/react';
 import ResendEmailButtonWrapper from '../resend-email-button-wrapper';
-
-jest.mock('Stores/connect.js', () => ({
-    __esModule: true,
-    default: 'mockedDefaultExport',
-    connect: () => Component => Component,
-}));
+import { StoreProvider } from '../../../hooks';
 
 describe('<ResendEmailButtonWrapper/>', () => {
-    const resendVerificationEmail = jest.fn();
+    let mockRootStore;
```

Comment:
```suggestion
    let mockRootStore: DeepPartial<TRootStore>;
```

Reply:
@farzin-fs By assigning the type I won't be able to make any change in`mockRootStore` hence I need to pass new amounts during the test, so I didn't assign type here

Reply:
@Bahar-FS Please ignore this comment 🙏🏻 
We are discussing this [here](https://deriv-group.slack.com/archives/C023SRL88VB/p1666772484650939) 🙇🏻